docs/develop/dotnet/message-passing.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: message-passing
+title: Application message passing - .NET SDK feature guide
```

Comment:
"Workflow interaction" I think is a better name for these concepts than "Application message passing"